lib/service.js

```diff
@@ -93,26 +88,31 @@ class Service {
     });
   }
 
+  static _checFormData(formData) {
```

Comment:
Extra function not needed, just add validation into `upload` method itself. 
Keep the interface of the `Service` class as clean/minimal as possible without extra methods - even if its an underscore/private method.